ui/app/components/app/modals/edit-approval-permission/edit-approval-permission.container.js

```diff
@@ -0,0 +1,26 @@
+import { connect } from 'react-redux'
+import { compose } from 'recompose'
+import withModalProps from '../../../../helpers/higher-order-components/with-modal-props'
+import EditApprovalPermission from './edit-approval-permission.component'
+import { resetAccount } from '../../../../store/actions'
+import { getSelectedIdentity } from '../../../../selectors/selectors'
+
+const mapStateToProps = (state) => {
+  const modalStateProps = state.appState.modal.modalState.props || {}
+  return {
+    selectedIdentity: getSelectedIdentity(state),
+    ...modalStateProps,
+  }
+}
+
+
+const mapDispatchToProps = dispatch => {
+  return {
+    resetAccount: () => dispatch(resetAccount()),
```

Comment:
I'm guessing you forgot to update this part

Reply:
This still looks wrong - `resetAcocunt` isn't sued in the `EditApprovalPermission` component